README.md

```diff
@@ -12,6 +12,62 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not define multiple types (ie class, struct, enum...) in the same file; each type should have its own file.
```

Comment:
I guess it should be more clear cause you add enum to the list of stuff to not include then down there it's in the list of stuff to include..